packages/server/src/api/routes/tests/viewV2.spec.ts

```diff
@@ -3865,6 +3866,45 @@ if (descriptions.length) {
               expect(rows[0].count).toEqual(2)
             })
 
+            isInternal &&
+              it("should be able to max a static formula field", async () => {
+                const table = await config.api.table.save(
+                  saveTableRequest({
+                    schema: {
+                      string: {
+                        type: FieldType.STRING,
+                        name: "string",
+                      },
+                      formula: {
+                        type: FieldType.FORMULA,
+                        name: "formula",
+                        formulaType: FormulaType.STATIC,
+                        responseType: FieldType.NUMBER,
+                        formula: "{{ string }}",
+                      },
+                    },
+                  })
+                )
+                await config.api.row.save(table._id!, {
+                  string: "1",
+                })
```

Comment:
Could you create more than 1 row for this test, just to make sure it is actually maxing something and not passing by some fluke?